modules/study/src/main/StudyRepo.scala

```diff
@@ -132,7 +132,7 @@ final class StudyRepo(private[study] val coll: AsyncColl)(using
           $set(
             "position"    -> s.position,
             "name"        -> s.name,
-            "flair"       -> s.flair,
+            "flair"       -> s.flair.pp("set flair in repo"),
```

Comment:
It seems that when `s.flair` is `None`, the field is not updated, is this correct or is the bug somewhere else? I could only find the `$setsAndUnsets` utility, but it's taking raw Bdoc and not ElementProducer.

It seems strange to me such a basic utility wouldn't have a higher level version, so maybe I'm missing something